src/frontend/Player/createPlayer.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  VideoPlayerInterface,
+  VideoPlayerType,
+} from 'types/VideoPlayerInterface';
+import { createPlyrPlayer } from './createPlyrPlayer';
+
+export type createPlayer = (
```

Comment:
this is pretty confusing imo. Either this should have a different name and a generic shape, or you could just fo with the function and use `typeof createPlayer` whenever you need the type.